packages/store/ts/config/v2/store.ts

```diff
@@ -120,5 +120,9 @@ export function resolveStoreConfig<const input>(input: validateStoreConfig<input
   } as resolveStoreConfig<input>;
 }
 
-// TODO(alvrs): swap with a better fully resolved type
-export type ResolvedStoreConfig = resolveStoreConfig<StoreConfigInput>;
+export type ResolvedStoreConfig = {
+  readonly tables: Dict<string, ResolvedTableConfig>;
+  readonly userTypes: UserTypes;
+  readonly enums: Dict<string, ReadonlyArray<string>>;
```

Comment:
```suggestion
  readonly enums: Dict<string, readonly [string, ...string[]]>;
```
is this any better/worse?